src/lib/plugins/world.js

```diff
@@ -41,6 +43,10 @@ module.exports.server = async function (serv, { version, worldFolder, generation
   serv.netherworld = new World(generations.nether(generation.options))
   // serv.endworld = new World(generations["end"]({}));
 
+  // todo as i understand this is only thing that should block connection
+  serv.emit('worldsReady')
```

Comment:
Please add to doc

Reply:
To address the comment:
I think its better to add `pluginsReady` property and event so we can create other async plugins that should block connections later (for now only builtin world.js is async). However I'm not sure what to throw in case if player connects too early like `Plugins are still initializing, try again later`?

Reply:
Done. see latest changes